Configurations.md

```diff
@@ -2747,7 +2747,7 @@ See also [`max_width`](#max_width) and [`use_small_heuristics`](#use_small_heuri
 
 ## `style_edition`
 
-Controls the edition of the [Rust Style Guide] to use for formatting ([RFC 3338])
+Controls the edition of the [Rust Style Guide](https://doc.rust-lang.org/nightly/style-guide/editions.html?highlight=editions#rust-style-editions) to use for formatting ([RFC 3338])
```

Comment:
Please revert this change. This overrides the link to the `Style Guide` defined below. Add a new link for the style editions in a new sentence..